plugin.program.dnsleaktest/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.program.dnsleaktest"
+version="1.1"
+name="DNS Leak-Test"
+provider-name="Space2Walker">
+<requires>
+  <import addon="xbmc.python" version="2.26.0"/>
+  <import addon="script.module.certifi" version="2017.07.27.1"/>
```

Comment:
You only need the `requests` import, not others. These are dependencies of the `requests` module.